MTA-7135: Agent Mode Compilation Guardrails AST Verification (#380)
migtools/mta-documentation: docs/topics/developer-lightspeed/ref_agent-mode-validation-checks.adoc

// Module included in the following assemblies:
//
// * docs/topics/developer-lightspeed/assembly_understanding_dl.adoc

:_mod-docs-content-type: REFERENCE
[id="agent-mode-validation-checks_{context}"]
= Agent mode validation checks

[role="_abstract"]

> In agent mode, the {mta-dl-plugin} performs specific verification checks on every code suggestion to ensure code validity.

Reviewer: WDYT?
```suggestion
To understand how the system ensures code validity, review the specific verification checks the {mta-dl-plugin} performs in agent mode. 
```
